JavaScript/JQuery/Internal.hs

```diff
@@ -3,55 +3,54 @@
-foreign import javascript unsafe "jQuery($1)"            jq_select            :: JSString                       -> IO JQuery
-foreign import javascript unsafe "jQuery()"              jq_selectEmpty       ::                                   IO JQuery
-foreign import javascript unsafe "jQuery($1,$2)"         jq_selectWithContext :: JSString -> JSVal              -> IO JQuery
-foreign import javascript unsafe "$2.css($1)"            jq_getCss            :: JSString             -> JQuery -> IO JSString
-foreign import javascript unsafe "$3.css($1,$2)"         jq_setCss            :: JSString -> JSString -> JQuery -> IO JQuery
-foreign import javascript unsafe "$1.height()"           jq_getHeight         ::                         JQuery -> IO Double
-foreign import javascript unsafe "$2.height($1)"         jq_setHeight         :: Double               -> JQuery -> IO JQuery
-foreign import javascript unsafe "$1.width()"            jq_getWidth          ::                         JQuery -> IO Double
-foreign import javascript unsafe "$2.width($1)"          jq_setWidth          :: Double               -> JQuery -> IO JQuery
-foreign import javascript unsafe "$1.innerHeight()"      jq_getInnerHeight    ::                         JQuery -> IO Double
-foreign import javascript unsafe "$1.innerWidth()"       jq_getInnerWidth     ::                         JQuery -> IO Double
-foreign import javascript unsafe "$2.outerHeight($1)"    jq_getOuterHeight    :: Bool                 -> JQuery -> IO Double
-foreign import javascript unsafe "$2.outerWidth($1)"     jq_getOuterWidth     :: Bool                 -> JQuery -> IO Double
-foreign import javascript unsafe "$2.trigger($1)"        jq_trigger           :: JSString             -> JQuery -> IO ()
-foreign import javascript unsafe "$2.triggerHandler($1)" jq_triggerHandler    :: JSString             -> JQuery -> IO ()
-foreign import javascript unsafe "$1.scrollLeft()"       jq_getScrollLeft     ::                         JQuery -> IO Double
-foreign import javascript unsafe "$2.scrollLeft($1)"     jq_setScrollLeft     :: Double               -> JQuery -> IO JQuery
-foreign import javascript unsafe "$1.scrollTop()"        jq_getScrollTop      ::                         JQuery -> IO Double
-foreign import javascript unsafe "$2.scrollTop($1)"      jq_setScrollTop      :: Double               -> JQuery -> IO JQuery
-foreign import javascript unsafe "$2.stop($1)"           jq_stop              :: Bool                 -> JQuery -> IO JQuery
-foreign import javascript unsafe "$1.focus()"            jq_focus             ::                         JQuery -> IO JQuery
+foreign import javascript unsafe "jQuery(document).ready($1)"  jq_ready             ::                     Callback a -> IO ()
+foreign import javascript unsafe "$2.addClass($1)"             jq_addClass          :: JSString             -> JQuery -> IO JQuery
```

Comment:
Other than the `document.ready` line, this is a whitespace change, right?  I want to make sure I'm not missing something subtle.